compiler/src/iree/compiler/Codegen/Common/GPU/test/gpu_vector_distribution.mlir

```diff
@@ -0,0 +1,24 @@
+// RUN: iree-opt -iree-transform-dialect-interpreter --split-input-file --cse %s | FileCheck %s
+
+#layout = #iree_vector_ext.layout<<[VECTORY, LANEY], [4, 4]>, <[VECTORX, LANEX], [4, 4]>>
+
+builtin.module attributes { transform.with_named_sequence } {
+  // CHECK-LABEL: @distribute_elementwise
+  func.func @distribute_elementwise(%a: vector<16x16xf16>, %b: vector<16x16xf16>) -> vector<16x16xf16> {
+    %c0 = arith.constant 0 : index
+    %cst_0 = arith.constant 0.0 : f16
+    // CHECK: %[[ROOT:.*]] = arith.constant dense<0.000000e+00> : vector<4x4xf16>
+    %root = arith.constant {"__vector_layout_test_anchor_result_0" = #layout} dense<0.0> : vector<16x16xf16>
+    // CHECK: %[[C:.*]] = arith.mulf %{{.*}}, %{{.*}} : vector<4x4xf16>
```

Comment:
Why do we create named matches but don't check the usage? I'd expect this to either not created named matches or check the usage in `iree_vector_ext.to_*` ops

Reply:
Fixed